examples/grafana/README.md

```diff
@@ -0,0 +1,31 @@
+# grafana-ocp
+
+This template creates a custom Grafana instance preconfigured to gather Prometheus openshift metrics.
+It is uses OAuth token to login openshift Prometheus.
+
+
+## To deploy grafana
+Note: make sure to have openshift prometheus deployed.
+(https://github.com/openshift/origin/tree/master/examples/prometheus)
+
+1. ```oc create project grafana```
+2. ```oc new-app -f grafana-ocp.yaml```
+3. ```oc expose svc grafana-ocp```
+4. grab the grafana url ``` oc get route |awk 'NR==2 {print $2}' ```
+5. grab the ocp token, from openshift master run: ```oc sa get-token management-admin -n management-infra```
```

Comment:
This command doesn't make sense.  What is management-infra?

Did you create a user called management-infra and a namespace called management-infra?  What about the grafana namespace?

Reply:
@jeremyeder 

IIUC, the grafana server does not use access authentication.

This command is a way to get the admin token for the prometheus server, using the prom-proxy [1] authentication proxy.

note:
a. running a playbook using openshift-ansible will create this sa and namespace
b. when installing prometheus using the prometheus.yaml  example (line 31 in reademe) it will use this token to authenticate.

[1] https://github.com/openshift/origin/blob/master/examples/prometheus/prometheus.yaml#L113

Reply:
p.s.

alternatively you can use `kube-system` namespace and the `prometheus`  sa

Reply:
but this (using `kube-system` namespace) will not work with promethus install using openshift-ansible [1]

[1] https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/defaults/main.yaml#L5

Reply:
will change that